lineapy/graph_reader/artifact_collection.py

```diff
@@ -45,11 +46,11 @@ class ArtifactCollection:
 
     def __init__(
         self,
-        artifacts: List[Union[str, Tuple[str, int]]],
+        artifacts: Union[
+            List[str], List[Tuple[str, int]], List[Union[str, Tuple[str, int]]]
+        ],
```

Comment:
FYI: This change is to address `mypy`'s type-check issue (`List` is [invariant](https://mypy.readthedocs.io/en/stable/common_issues.html#invariance-vs-covariance))